### PR TITLE
[PLUGIN-1097] add bigdecimal to BQ Table batch sink/source markdown

### PR DESCRIPTION
**Pull request by sumengwang**
# [PLUGIN-1097] add bigdecimal to BQ Table batch sink/source markdown

## Links
[PLUGIN-1097](https://cdap.atlassian.net/browse/PLUGIN-1097)

### CODE REVIEW

**Review by sechegaray — APPROVED** [review]
LGTM

File: docs/BigQueryMultiTable-batchsink.md [hunk 1]
```diff
@@ -90,6 +90,34 @@ When running on other clusters, the file must be present on every node in the cl
...
+
+| CDAP type      | BigQuery type                               |
+|----------------|---------------------------------------------|
+| array          | repeated                                    |
+| boolean        | bool                                        |
+| bytes          | bytes                                       |
+| date           | date                                        |
+| datetime       | datetime, string                            |
+| decimal        | numeric                                     |
+| bigdecimal     | bignumeric                                  |
```
Comment (rrielley76) [inline docs/BigQueryMultiTable-batchsink.md:106]: CDAP=decimal

File: docs/BigQueryTable-batchsink.md [hunk 2]
```diff
@@ -150,23 +150,24 @@ Data Type Mappings from CDAP to BigQuery
...
-| union          | unsupported   |
+| CDAP type      | BigQuery type                               |
+|----------------|---------------------------------------------|
+| array          | repeated                                    |
+| boolean        | bool                                        |
+| bytes          | bytes                                       |
+| date           | date                                        |
+| datetime       | datetime, string                            |
+| decimal        | numeric                                     |
+| bigdecimal     | bignumeric                                  |
```
Comment (rrielley76) [inline docs/BigQueryTable-batchsink.md:161]: CDAP = decimal

File: docs/BigQueryTable-batchsource.md [hunk 3]
```diff
@@ -89,19 +89,20 @@ Data Type Mappings from BigQuery to CDAP
...
+|---------------|---------------------------------------------|
+| bool          | boolean                                     |
+| bytes         | bytes                                       |
+| date          | date                                        |
+| datetime      | datetime, string                            |
+| float64       | double                                      |
+| geo           | unsupported                                 |
+| int64         | long                                        |
+| numeric       | decimal (38 digits, 9 decimal places)       |
+| bignumeric    | bigdecimal                                  |
```
Comment (rrielley76) [inline docs/BigQueryTable-batchsource.md:102]: CDAP = decimal

**Pull request merged**